library/Mockery/Exception/NoMatchingExpectationException.php

```diff
@@ -20,17 +20,84 @@
 
 namespace Mockery\Exception;
 
+use Hamcrest\Util;
 use Mockery;
+use SebastianBergmann\Comparator\ComparisonFailure;
```

Comment:
Could you please add the Comparator package then as required dependency to our composer file? Mockery can be used without phpunit, so I'm afraid in those cases this dependency would be most likely missing.